region/norfair/west/Crumble Shaft.json

```diff
@@ -354,6 +354,26 @@
         "Climb up 3 screens, and perform a turnaround buffered spin-jump away from the door to trigger the transition, bypassing any lock on the door."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Grapple Teleport",
+      "entranceCondition": {
+        "comeInWithGrappleTeleport": {
+          "blockPositions": [[5, 3]]
+        }
+      },
+      "requires": [
+        {"heatFrames": 100},
```

Comment:
```suggestion
        {"heatFrames": 120},
```

Reply:
And there should be a version without offscreenmovement

Reply:
Done.